src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ProgrammingExerciseServerSideTask } from 'app/entities/hestia/programming-exercise-task.model';
+import { FeedbackDetail } from 'app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component';
+
+export interface SimplifiedTask {
+    taskName: string;
+    testCases: ProgrammingExerciseServerSideTask['testCases'];
+}
+
+export interface FeedbackDetailsWithResultIdsDTO {
+    feedbackDetails: FeedbackDetail[];
+    resultIds: number[];
+}
+
+@Injectable()
+export class FeedbackAnalysisService {
+    private readonly resourceUrl = 'api/programming-exercises';
+    private readonly exerciseResourceUrl = 'api/exercises';
```

Comment:
Follow-up: According to [https://docs.artemis.cit.tum.de/dev/guidelines/client.html](https://docs.artemis.cit.tum.de/dev/guidelines/client.html), `readonly` variables should be in screaming snake case.